src/rviz_visual_tools.cpp

```diff
@@ -81,6 +81,14 @@ bool RvizVisualTools::deleteAllMarkers()
   return publishMarker(reset_marker_);
 }
 
+bool RvizVisualTools::deleteAllMarkers(const std::string& ns)
+{
+  visualization_msgs::msg::Marker delete_ns_marker = reset_marker_;
+  delete_ns_marker.header.stamp = builtin_interfaces::msg::Time();
```

Comment:
`builtin_interfaces::msg::Time` & `visualization_msgs::msg::Marker` are for ROS2 and which isn't the case on the master branch

Reply:
oops, good catch. should be fixed now